README.md

```diff
@@ -1 +1,236 @@
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hello-Microverse <a name="about-project"></a>
+> I created Hello-microverse using HTML and CSS. This is my first milestone at microverse.
+
+## 🛠 Built With <a name="built-with"></a>
+
+- HTML
+- CSS
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+    <li> CSS</li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- Added index.html flle.
+- Added style.css file.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://google.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+1.Web browser.
+
+2.Code editor.
+
+
+### Setup
+Clone this repository to your desired folder:
+
+Run this commands:
+
+```sh
+  
+  git clone git@github.com:Graceshiks/Hello-microverse.git
+```
+```sh
+  cd Hello-microverse
+   
+```
+
+### Install
+
+Install this project with:
+
+
+
+### Usage
+
+To run the project, execute the following command:
+
+>Open index.html using live server.
+
+### Run tests
+
+>coming soon
+
+
+### Deployment
+
+
+> coming soon
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+>
+
+👤 Grace Kamau
+
+- GitHub: [Github](https://github.com/Graceshiks)
+- Twitter: [Twitter](https://twitter.com/GrakamauGrace)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/grace-wanjiku-96aaa3109/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] Add Javascript.
+- [ ] Add more styling.
+- [ ] Add add header and footer.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](git@github.com:Graceshiks/Hello-microverse/issues/).
+
```

Comment:
- [ ] Kindly fix the issues page link since it is broken, you can use this one which will redirect the user to the issues page.
```suggestion

Feel free to check the [issues page](../../issues/).

```